src/app.rs

```diff
@@ -292,6 +300,31 @@ where
             states.stop(StateData::new(world, &mut self.data));
         }
 
+        // Read the Trans queue and apply changes.
+        {
+            /*let trans = {
+                let mut v = self.world.write_resource::<TransQueue<T, E>>();
+                let x = v.drain(..).collect::<VecDeque<_>>();
+                x
+            };*/
+            let mut world = &mut self.world;
+            let states = &mut self.states;
+            let reader = &mut self.trans_reader_id;
+
+            /*let trans = world.exec(|ev: Read<EventChannel<TransEvent<T, E>>>| {
```

Comment:
Clean comment?
Here and everywhere else

Reply:
Yeah, commented code is probably not updated, better remove it.